l2gersync/processor.go

```diff
@@ -198,7 +175,7 @@ func (p *processor) GetFirstGERAfterL1InfoTreeIndex(
 	ctx context.Context, l1InfoTreeIndex uint32) (GlobalExitRootInfo, error) {
 	e := GlobalExitRootInfo{}
 	err := meddler.QueryRow(p.database, &e, `
-		SELECT l1_info_tree_index, global_exit_root
+		SELECT l1_info_tree_index, block_num, global_exit_root
```

Comment:
Are you missing block position?

Reply:
Yes, thank you. Fixed https://github.com/agglayer/aggkit/pull/743/commits/67a1d9a621de698f9a4831e45baf85dbc47cb42f